scripts/check_version

```diff
@@ -3,29 +3,34 @@
 PACKAGE_VERSION=`grep '"version":' ./package.json | cut -d '"' -f4`
 ANDROID_GRADLE_VERSION=`grep "versionName " ./android/app/build.gradle | cut -d '"' -f2`
 TEMP_IOS_VAR=`perl -n -e '/MARKETING_VERSION = (.*);/ && print "$1\n"' ./ios/HathorMobile.xcodeproj/project.pbxproj`
-IOS_PROJECT_VERSION_1=`echo $TEMP_IOS_VAR | awk '{print $1}'`
-IOS_PROJECT_VERSION_2=`echo $TEMP_IOS_VAR | awk '{print $2}'`
+IOS_MARKETING_VERSION_1=`echo $TEMP_IOS_VAR | awk '{print $1}'`
+IOS_MARKETING_VERSION_2=`echo $TEMP_IOS_VAR | awk '{print $2}'`
+
+# This will read everything of the string up to the first dash.
+STRIPPED_PACKAGE_VERSION=$(echo $PACKAGE_VERSION| cut -d- -f1);
 
 # For debugging:
-# echo x${SRC_VERSION}x
-# echo x${ELECTRON_VERSION}x
-# echo x${PACKAGE_VERSION}x
+# echo package: x${PACKAGE_VERSION}x
+# echo stripped: x${STRIPPED_PACKAGE_VERSION}x
+# echo android: x${ANDROID_GRADLE_VERSION}x
+# echo ios pv1: x${IOS_MARKETING_VERSION_1}x
+# echo ios pv2: x${IOS_MARKETING_VERSION_2}x
 
 EXITCODE=0
 
 if [[ x${PACKAGE_VERSION}x != x${ANDROID_GRADLE_VERSION}x ]]; then
-	echo Version different in package.json and android/app/build.gradle
+	echo Version different in package.json \($PACKAGE_VERSION\) and android/app/build.gradle
 	EXITCODE=-1
 fi
 
-if [[ x${PACKAGE_VERSION}x != x${IOS_PROJECT_VERSION_1}x ]]; then
-	echo Version different in package.json and first MARKETING_VERSION on ios/HathorMobile.xcodeproj/project.pbxproj
+if [[ x${STRIPPED_PACKAGE_VERSION}x != x${IOS_MARKETING_VERSION_1}x ]]; then
+  echo Version different in package.json \($PACKAGE_VERSION\) and first MARKETING_VERSION \($IOS_MARKETING_VERSION_1\) on ios/HathorMobile.xcodeproj/project.pbxproj
 	EXITCODE=-1
 fi
 
-if [[ x${PACKAGE_VERSION}x != x${IOS_PROJECT_VERSION_2}x ]]; then
-	echo Version different in package.json and second MARKETING_VERSION on ios/HathorMobile.xcodeproj/project.pbxproj
+if [[ x${STRIPPED_PACKAGE_VERSION}x != x${IOS_MARKETING_VERSION_2}x ]]; then
+	echo Version different in package.json \($PACKAGE_VERSION\) and second MARKETING_VERSION \($IOS_MARKETING_VERSION_2\) on ios/HathorMobile.xcodeproj/project.pbxproj
 	EXITCODE=-1
 fi
 
```

Comment:
Can we add a new check here if the `CURRENT_PROJECT_VERSION` is in the correct format `X.Y.Z`?

If we are going to throw error in the About page, we must guarantee that this version is correct here.

Reply:
Done